docs/pages/_app.js

```diff
@@ -55,7 +55,7 @@ function LanguageNegotiation() {
         acceptLanguage.get(navigator.language) ||
         userLanguage;
```

Comment:
Should we redirect to translated page if navigator language is not English though?
Given that content is not translated, I would argue that we should only redirect if user explicitly selected different language

```diff
const preferedLanguage =
  LANGUAGES.find((lang) => lang === getCookie('userLanguage')) ||
- acceptLanguage.get(navigator.language) ||
  userLanguage;
```

Reply:
The pages for the data grid are not translated, but the pages for the core components are. Speaking about Portuguese, the core docs is around 35% translated. I think as soon as one single string is translated we'll need to redirect the docs. BTW, the button to send contributions doesn't work on X. I'll open an issue for that.

![image](https://user-images.githubusercontent.com/42154031/169532618-fbd542b5-b0d8-4a29-9138-632a14bd7822.png)

Reply:
> I think as soon as one single string is translated we'll need to redirect the docs.

Interesting, what is the motivation behind this?
I would argue that partially translated (even >50%) page is more disappointing than helpful. Especially for technical documentation (landing pages and pricing page are English-only)